Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -0,0 +1,20 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class CommentCell: UITableViewCell, ValueCell {
```

Comment:
This class will get replaced entirely once this [PR](https://github.com/kickstarter/ios-oss/pull/1480) goes in.

Reply:
The above PR also contains tests for this class.